app/controllers/ops_controller/diagnostics.rb

```diff
@@ -720,10 +694,6 @@ def diagnostics_set_form_vars
     end
   end
 
-  def orphaned_records_get
-    @sb[:orphaned_records] = MiqReportResult.orphaned_counts_by_userid
```

Comment:
This `MiqReportResult.orphaned_counts_by_userid` seems to be only called here. See https://github.com/search?q=org%3AManageIQ%20orphaned_counts_by_userid&type=code .  Can you also delete this in the core repo?